distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -183,8 +183,16 @@ grant codeBase "file:/pax-logging-log4j2" {
     permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}solr.log", "read,write";
     permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}artemis.log", "read,write";
     permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}security.log", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}securityBackup.log", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}log4j2.config.xml", "read";
     permission java.io.FilePermission "pax-logging.properties", "read";
     permission java.net.SocketPermission "*", "listen,resolve";
+
+    //  Security-Hardening: uncomment the following line and replace <PATH> with
+    //  the path to the new desired log file
+
+    //  permission java.io.FilePermission "<PATH>", "read,write";
```

Comment:
❓ Should we put this in configurations.policy? I'd like to make sure we are consistent on how we want to handle permissions related to configuration. @stustison Do you have any suggestions?

Reply:
I think I agree, the configuration policy file seems like the correct place for this.

Reply:
I think thats a better place for it. I agree and changed :+1: